src/App.tsx

```diff
@@ -1,24 +1,274 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { prepareTodos } from './utils/prepareTodos';
+import { TodoList } from './components/TodoList';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  callAddTodo,
+  callDeleteTodo,
+  callEditTodo,
+  callGetTodos,
+} from './api/todos';
+import { ErrorType } from './types/ErrorType';
+import { FilterBy } from './types/FilterBy';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { RequestTodo } from './types/RequestTodo';
+import { Notifications } from './components/Notificiations';
 
-const USER_ID = 0;
+const USER_ID = 6414;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.LOAD);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ALL);
+  const [hasError, setHasError] = useState(false);
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todoWithLoader, setTodoWithLoader] = useState(0);
```

Comment:
it's not clear why `todoWithLoader` contains some number
number can represent `id` value or something else
but in this variable I expect to see some `todo`
let's think more about variable name